src/AppBundle/Entity/Repository/ReportSubmissionRepository.php

```diff
@@ -112,28 +112,23 @@ public function findDownloadableOlderThan(\DateTime $olderThan, $limit)
     }
 
     /**
-     * @param string $status        string new|archived
-     * @param string $q             serach string
-     * @param string $createdByRole see values in USER::ROLE_*
-     * @param int    $offset
-     * @param int    $limit
-     * @param string $orderBy       default createdOn
-     * @param string $order         default ASC
-     *
-     * @return ReportSubmission[]
+     * @param $offset
+     * @param $limit
+     * @param array $fromDate
+     * @param array $toDate
+     * @param string $orderBy default createdOn
+     * @param string $order default ASC
+     * @return array
      */
     public function findAllReportSubmissions(
-        $status,
-        $q,
-        $createdByRole,
         $offset,
         $limit,
+        array $fromDate,
```

Comment:
any reason why the DateTime is not passed directly here (default null) ?

Reply:
Fair point. I've updated the signature to accept DateTime objects.